app/ports/templates/ports/layout.html

```diff
@@ -49,6 +49,47 @@
     {% block content %}
     {% endblock %}
 </div>
+<br><br>
+<div class="bg-secondary">
+    <div class="container p-4 text-light">
+        <div class="row">
+            <div class="col-lg-4">
+                <strong>
+                    Quick Links:</strong>
+                    <ul>
+                        <li><a class="footer-links" href="{% url 'about_page' %}">About</a></li>
+                        <li><a class="footer-links" href="{% url 'stats_faq' %}">Statistics FAQ</a></li>
+                        <li><a class="footer-links" href="https://github.com/macports/macports-webapp">Git Repository</a></li>
+                        <li><a class="footer-links" href="https://github.com/macports/macports-webapp/tree/master/docs">Documentation</a></li>
+                        <li><a class="footer-links" href="https://github.com/macports/macports-webapp/tree/master/docs/api_v1">API</a></li>
+                        <li><a class="footer-links" href="https://docs.google.com/document/d/1TqNHm3NrtLDelWqY7P4pZyTMZDJrlsPymFZklzV2t24/edit">Feature Requests</a></li>
+                        <li><a class="footer-links" href="https://github.com/macports/macports-webapp/issues">Issues</a></li>
+                    </ul>
+            </div>
+            <div class="col-lg-4">
+                <strong>MacPorts</strong>
+                <ul>
+                    <li><a class="footer-links" href="https://www.macports.org">MacPorts Home</a></li>
+                    <li><a class="footer-links" href="https://www.macports.org/install.php">Install MacPorts</a></li>
+                    <li><a class="footer-links" href="https://trac.macports.org/wiki/FAQ">MacPorts FAQ</a></li>
+                    <li><a class="footer-links" href="https://www.macports.org/news/">MacPorts NEWS</a></li>
+                </ul>
+            </div>
+            <div class="col-lg-4">
+                Port Information was last updated at:<br>
+                <i class="footer-links">{{ port_info_updated_at|date:'Y-m-d H:i' }} (UTC)</i><br>
+                <i><a class="footer-links" target="_blank" href="https://github.com/macports/macports-ports/commit/{{ port_info_updated_till_commit }}/">{{ port_info_updated_till_commit|slice:":8" }}</a></i><br><br>
```

Comment:
I was thinking of this:
```suggestion
                <i><a class="footer-links" target="_blank" href="https://github.com/macports/macports-ports/commits/{{ port_info_updated_till_commit }}/">{{ port_info_updated_till_commit|slice:":8" }}</a></i><br><br>
```
or even better something along these lines
```suggestion
                <i><a class="footer-links" target="_blank" href="https://github.com/macports/macports-ports/commits/master?after={{ port_info_updated_till_commit }}+1">{{ port_info_updated_till_commit|slice:":8" }}</a></i><br><br>
```
But I cannot get the second one working as desired, it's always off by at least one; I guess it wasn't meant to be used like this.

And keys listed on https://developer.github.com/v3/repos/commits/ don't help either.

My idea was to show the list of the last few commits rather than just a single one.